manifests/e/Eraser/Eraser/6.2.0.2992/Eraser.Eraser.locale.en-US.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate
+
+PackageIdentifier: Eraser.Eraser
+PackageVersion: 6.2.0.2992
+PackageLocale: en-US
+Publisher: Eraser
+PackageName: Eraser

```

Comment:
```suggestion
PackageName: Eraser
```

Reply:
![image](https://user-images.githubusercontent.com/72118300/120938691-f9b3c880-c6c8-11eb-9b3a-fb4e0d9a324d.png)
it legit doesn't let me commit the suggestion

Reply:
Dafak, this most be `Eraser.Eraser`

Reply:
?

Reply:
Nevermind i'm sleepy, i don't think you're allowed because its the exact same text

Reply:
Yeah I guess that's probably it.